Week1/QA_PREP_EXERCISE.md

```diff
@@ -0,0 +1,17 @@
+As a preparation step for the upcoming Q&A, you will work through the following exercise. The expected result is a list of SQL `CREATE TABLE` statements that represent the tables you're being asked to design. We suggest you first think of the design of the tables (names, columns, types) and then proceed to write the SQL statements, that way you are thinking about the problem you're trying to solve (creating a database for a website), rather than the implementation first (creating a set of SQL statements).
+
+### Option 1
+
+Design the tables for a database that represents a website with food recipes. You don't need to write any specific queries at the moment, but the design should consider the need for some future queries.
+
+- Each recipe should have a name, a category, a list of ingredients, and a list of steps to follow to complete the recipe. Many recipes might share the same ingredients or the same list of steps (For example, cook pasta according to the package instructions).
+- The website will show a general page with all the recipes.
+- The website will also show recipes under a single category (Salads, Mexican, etc).
+
```

Comment:
Maybe we can add some questions here, like:
1. Which entities can you spot in the above problem ? 
2. Which tables do you need to create to store the above data?
3. What are the relationships between those entities?

Reply:
Out of these 3, I think the first two questions are the most relevant and answerable for week 1.

Reply:
I'd leave the 3rd one though, this is exactly what we're trying to hint at, getting them to think about something called "relationships" even if they haven't formally studied it.